.authors.yml

```diff
@@ -1,6 +1,6 @@
 - name: Michael Sarahan
   email: msarahan@continuum.io
-  num_commits: 665
+  num_commits: 1330
```

Comment:
unfortunately the unpatched rever multi-counts commits from aliased contributors.